source/js/controllers/search_controller.js

```diff
@@ -46,13 +46,24 @@ app.Controllers.SearchController = Marionette.Controller.extend({
 
     if(cad === 'nearest') {
       this.searchByLocation(this.page);
-    } else if(/^\d{5}(-\d{4})?$/.test(cad)) {
-      this.searchByZipcode(cad, this.page);
     } else {
-      this.searchByName(cad, this.page);
```

Comment:
are you still using searchByName? can that get deleted too? (not seeing it referenced in search_controller.js)